tests/nnstreamer_filter_tvm/unittest_filter_tvm.cc

```diff
@@ -393,8 +393,10 @@ TEST (nnstreamerFilterTvm, invoke00)
   };
 
   output.size = input.size = sizeof (float) * 3 * 640 * 480 * 1;
-  input.data = g_malloc (input.size);
-  output.data = g_malloc (output.size);
+
+  input.data = aligned_alloc((size_t)128, input.size);
```

Comment:
Q) Is the number 128 not related to `GST_TENSOR_ALLOC_ALIGNMENT`?

Reply:
Yes it is. `GST_TENSOR_ALLOC_ALIGNMENT` is given as bitmask so this also can be written as `GST_TENSOR_ALLOC_ALIGNMENT`+1